LoopKit/GlucoseUnitPublishing.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  GlucoseUnitPublishing.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2021-01-13.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+
+public protocol GlucoseUnitObserver {
```

Comment:
I wonder if we should change this name to something like "preferredGlucoseDisplayUnitObserver", just to keep us in the mindset that this does not represent a change in storage units, but rather how the user wants to see things. I wouldn't hold up this PR for this naming change, but it's a conflation I keep seeing.